components/DetailCard.js

```diff
@@ -1,13 +1,13 @@
-import { Container, Card, Header, Footer } from "../styles/styles";
-
+import { Container, Card, Header } from "../styles/styles";
 import Link from "next/link";
 import styled from "styled-components";
+import Image from "next/image";
 
 export default function DetailCard({ campingTrip }) {
   return (
     <>
       <Header>
-        <h1>Check the details of your trips</h1>
+        <h1>Check the details of your trip</h1>
       </Header>
```

Comment:
No. ImageUpload.js and FormCard.js are not pages and on the FormPage is only the FormCard, no Heading, Footer, Navigation Elements etc. You could refator this as well so that all your components are only for reusable components.